rpm-version-checker.sh

```
@@ -29,6 +26,48 @@ if ! command -v rpmdev-vercmp &>/dev/null; then
+
+# Compare a wanted version-release ($1) against an installed one ($2) and print
+# a coloured verdict. rpmdev-vercmp exits 11 when the first arg is newer, i.e.
+# the installed version is older and an upgrade is needed; 12 when installed is
+# newer; 0 when equal.
+report_status() {
+  set +o errexit
+  rpmdev-vercmp "$1" "$2" >/dev/null 2>&1
+  local exit_code="$?"
+  set -o errexit
+  if [[ $exit_code == 11 ]]; then
+    printf "$RED upgrade needed\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)